tests/consensus/test_soft_voided4.py

```diff
@@ -10,7 +13,11 @@
 class BaseSoftVoidedTestCase(SimulatorTestCase):
     seed_config = 5988775361793628169
 
-    def _run_test(self, simulator, soft_voided_tx_ids):
+    def _run_test(
+        self,
+        simulator: Simulator,
+        soft_voided_tx_ids: set[VertexId] | list[VertexId]
```

Comment:
I guess `soft_voided_tx_ids` must be a list. Maybe change it here and fix any wrong callers.

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/971/commits/eea7f836df7ecdbaedb30ae29fd4ca6ede81c3fc

Reply:
Thanks!